src/repr/src/proto/adt/numeric.rs

```diff
@@ -29,3 +29,19 @@ impl TryFrom<ProtoNumericMaxScale> for NumericMaxScale {
         Ok(NumericMaxScale(u8::from_proto(repr.value)?))
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::proto::protobuf_roundtrip;
+    use proptest::prelude::*;
+
+    proptest! {
+        #[test]
+        fn numeric_max_scale_protobuf_roundtrip(expect in any::<NumericMaxScale>()) {
```

Comment:
It looks all good to me, but I have a question: What is our guidelines for which we write proptest? This case would be caught by testing `ScalarType` roundtrip.

Reply:
I think we discussed that we want to have a single test for `A` if `A` depends on `B` and both are defined in the same file, and two separate tests if `A` depends on `B` but they are defined in different files.